src/dropzone.coffee

```diff
@@ -1177,7 +1177,8 @@ class Dropzone extends Emitter
 
     extend headers, @options.headers if @options.headers
 
-    xhr.setRequestHeader headerName, headerValue for headerName, headerValue of headers
```

Comment:
:coffee: :sparkles: `xhr.setRequestHeader headerName, headerValue for headerName, headerValue of headers when headerName `